sdcm/cluster.py

```diff
@@ -2022,9 +2022,7 @@ def install_scylla(self, scylla_repo):
             self.remoter.run('sudo apt-get install -y rsync')
             self.download_scylla_repo(scylla_repo)
             self.remoter.run('sudo apt-get update')
-            self.remoter.run(
-                'sudo apt-get install -y '
-                ' {} '.format(self.scylla_pkg()))
+            self.install_package(self.scylla_pkg())
```

Comment:
I don't think that's the reason, it's failing on the same thing you fixed before

ca-certificata-java is failing.

Reply:
hum... checking